scripts/build-packages.js

```diff
@@ -0,0 +1,26 @@
+const path = require("path");
+const fs = require("fs");
+const shell = require("shelljs");
```

Comment:
miss `shelljs` in `root package.json`

What is the reason why `child_process` is not used here?

http://nodejs.cn/api/child_process.html#child_process_child_process_execsync_command_options

Reply:
We need to log the building outputs subsequently. Otherwise outputs of different packages are all tangled.

Reply:
OK, Now you need to add shelljs to `root package.json`

Reply:
Done.